docs/reference/server-args.md

```diff
@@ -0,0 +1,18 @@
+---
+hide:
+  - toc
+
+title: Appium Server Arguments
+---
+
+Some driver arguments can be set when launching the Appium server. This can be done as follows:
+
+```
+appium --driver='{"xcuitest": {[argName]: [argValue]}}'
```

Comment:
this is no longer the way it works. now drivers expose their args and it gets bundled into what the server recognizes. in this case this doc should refer to:

```
--driver-xcuitest-webdriveragent-port
```

Reply:
Thanks - will change this 👍